### ISSUE TITLE
[Quest][TBC] 10211 City of Light

### ISSUE THREAD

**Issue by hollingsworthless** [dalaranwow/dalaran-wow#4530]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
During quest 10211 City of Light in Shattrath City, when following NPC 19685 Khadgar's Servant around the city, the NPC clips through a ramp and disappears making the quest uncompletable. **This quest is necessary to align with either the Scryers or Aldors in Shattrath.**

**Expected behaviour**: Tell us what should happen instead
The NPC should continue the tour of the city walking up the ramp and finishing the tour, ending back at Archmage Khadgar.

**Steps to reproduce the problem**:
1. Accept and complete quest 10210 "A'dal" from NPC 19684 Haggard War Veteran on bridge on north side of Shattrath. This quest is the first in the chain required to accept the next quest.
2.  Accept quest 10211 "City of Light" from NPC 18166 Archmage Khadgar.
3.  Follow NPC 19685 Khadgar's Servant that spawns upon accepting the quest around the tour until he reaches the base of the ramp on the very north side of the Terrace of Light.

**Include proofs for this behaviour**
Video shows entire tour from accepting the quest to the bug. Bug comes around 2:50 mark:
https://www.youtube.com/watch?v=WB5odwynhUg

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
This already works fine on PTR.

![GitHub Logo](http://i.imgur.com/akNZ6k9.jpg)
![GitHub Logo](http://i.imgur.com/v6Nq8OT.jpg)

**Comment by hollingsworthless** [comment]
When will the PTR go live? I see issues that are awaiting PTR update from 4 months ago?

**Comment by Rushor** [comment]
according to the news on the website in the next 1-3 months. i cannot give an exact date.

**Comment by hollingsworthless** [comment]
Thanks!
